### PR TITLE
Move requirements to requirements.txt

### PR DESCRIPTION
**Pull request by jeremycline**
This makes it easy to `pip install -r` or install the dependencies with
Ansible.

This commit also adds the missing dependency on fedmsg

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -43,8 +60,8 @@ def run_tests(self):
     url="https://github.com/fedora-infra/fmn.sse/",
     # download_url="https://pypi.python.org/pypi/fmn.sse/",# not yet configured
     license='GPL',
-    install_requires=requires,
-    tests_require=tests_require,
+    install_requires=get_requirements('requirements.txt'),
```
Comment (skrzepto) [inline setup.py]: Since its going to read the file instead

**Comment by skrzepto** [comment]
Besides that small comment about deleting the two variable this is :+1: LGTM

File: setup.py (hunk 1 above)
Comment (jeremycline) [inline setup.py]: Oops, good catch

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**